router.go

```diff
@@ -80,10 +80,11 @@ func (r *Router) PATCH(p string, h HandlerFunc) {
 // Group creates a new Router with a prefix for all routes
 func (r *Router) Group(p string) *Router {
 	return &Router{
-		mux:        r.mux,
-		prefix:     p,
-		routes:     Routes{},
-		middleware: r.middleware.Copy(),
+		mux:           r.mux,
+		prefix:        p,
+		routes:        Routes{},
+		middleware:    r.middleware.Copy(),
+		errorHandlers: r.errorHandlers,
```

Comment:
The new `Router` should not hold a reference to the "global" `ErrorHandlers`, make a `Copy` func just like middleware has.

There will be cases where you don't want the same `ErrorHandler` on all groups (e.g., on a `/api` group I want to return error message as JSON, but on `/pages` group I want to return a error page as HTML).